service/src/main/java/bio/terra/workspace/service/resource/controlled/ControlledResourceService.java

```diff
@@ -55,7 +58,10 @@
 @Component
 public class ControlledResourceService {
 
-  private static final int MAX_ASSIGNED_USER_LENGTH = 254;
+  // These are chosen to retry a maximum length so we return under a 30 second
```

Comment:
I'm lost how the math works out for the 30 seconds?

Reply:
I switched to using the timer, FYI.